modules/tor.sh

```diff
@@ -54,6 +54,32 @@ function tor {
 
     restart_service tor
     echo "Success: the port has been added"
+  elif [ "$1" = "delete" ]; then
```

Comment:
@Kid243 this is a branch from a branch

Reply:
@dogi not a branch from a branch, I copy pasted the code from delete1port into this one anticipating the possibility that delete1port might be merged first